packages/network/src/logic/Node.ts

```diff
@@ -101,13 +101,13 @@ export class Node extends EventEmitter {
         this.trackerNode = opts.protocols.trackerNode
         this.peerInfo = opts.peerInfo
 
-        this.connectToBootstrapTrackersInterval = opts.connectToBootstrapTrackersInterval || 5000
         this.sendStatusToAllTrackersInterval = opts.sendStatusToAllTrackersInterval || 1000
         this.bufferTimeoutInMs = opts.bufferTimeoutInMs || 60 * 1000
         this.bufferMaxSize = opts.bufferMaxSize || 10000
         this.disconnectionWaitTime = opts.disconnectionWaitTime || 30 * 1000
         this.nodeConnectTimeout = opts.nodeConnectTimeout || 15000
         this.instructionRetryInterval = opts.instructionRetryInterval || 60000
+        this.trackerConnectionMaintenanceInterval = opts.trackerConnectionMaintenanceInterval ?? 5000
```

Comment:
👍

Reply:
Hm, does this mean there could be up to a 5s delay before a node will connect to the stream-appropriate tracker, and thus up to a 5s delay before it will start connecting to peers on a different tracker?